api/src/main/java/org/apache/iceberg/TableScan.java

```diff
@@ -107,24 +68,6 @@ default TableScan select(String... columns) {
     return select(Lists.newArrayList(columns));
```

Comment:
couldn't move this select method to the `Scan` interface, as API compatibility check tool shows that the return type of `TableScan` is changed to `Scan` probably due to varargs.